codespell_lib/data/dictionary.txt

```diff
@@ -22468,6 +22468,7 @@ rescrition->restriction
 rescritions->restrictions
 reseach->research
 reseached->researched
+researved->reserved
```

Comment:
Do you want to add the reserves, reservation and reservations version of this typo too

Reply:
I added the suggested versions. Thanks